pkg/dib/build.go

```diff
@@ -121,7 +124,11 @@ func RebuildNode(
 
 	if img.NeedsTests {
 		buildReport.TestsStatus = report.TestsStatusPassed
+		buildReport.ScanStatus = report.ScanStatusPassed
 		if err := testImage(img, testRunners, dibReport); err != nil {
+			if errors.Is(err, trivy.ErrCommandFailed) {
```

Comment:
I think there should not be a mention of any specific implementation of the TestRunner abstraction here (`trivy.ErrCommandFailed`).

Maybe the `ErrCommandFailed` should be part of the same package that declared the `TestRunner` interface instead.